.github/workflows/test.yml

```diff
@@ -21,11 +21,11 @@ jobs:
           - windows-latest
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v4
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version-file: ${{ matrix.node-version }}
```

Comment:
You aren't passing a file name here, so it should still be `node-version`

Reply:
yeah I know, just messing around